packages/database/src/migrations/20210412055533-LaosEocEntitycleanUp-modifies-data.js

```diff
@@ -20,16 +20,218 @@ exports.up = async function (db) {
+    'FETP_iCAni20',
+    'FETP_iSHag86',
+    'FETP_iRJoe83',
+    'FETP_iJKep96',
+    'FETP_iCPid46',
+    'FETP_iEPun59',
+    'FETP_iCAqu76',
+    'FETP_iWBia32',
+    'FETP_iTAna94',
+    'FETP_iNTum45',
+    'FETP_iRBat22',
+    'FETP_iJSau36',
+    'FETP_iYMwa37',
+    'FETP_iGTow39',
+    'FETP_iMKae64',
+    'FETP_iKYam47',
+    'FETP_iSEli80',
+    'FETP_iRWak82',
+    'FETP_iTTun77',
+    'FETP_iLLak9',
+    'FETP_iRGat42',
+    'FETP_iGKun11',
+    'FETP_iEBen10',
+    'FETP_iATab93',
+    'FETP_iESta62',
+    'FETP_iINar40',
+    'FETP_iJAme78',
+    'FETP_iJTei31',
+    'FETP_iGSui30',
+    'FETP_iROak35',
+    'FETP_iBBom24',
+    'FETP_iGTen16',
+    'FETP_iPMuk33',
+    'FETP_iCJos79',
+    'FETP_iWMar7',
+    'FETP_iDKis66',
+    'FETP_iJKum6',
+    'FETP_iBPen12',
+    'FETP_iAKum48',
+    'FETP_iAMar43',
+    'FETP_iEHev19',
+    'FETP_iRRob60',
+  ];
+  const newEntityCodes = [
+    'FETP-3C3X',
+    'FETP-Q6IR',
+    'FETP-PEIX',
+    'FETP-DYO3',
+    'FETP-PHEF',
+    'FETP-CS2L',
+    'FETP-IO9M',
+    'FETP-WSLS',
+    'FETP-V6U5',
+    'FETP-IHNV',
+    'FETP-NKJM',
+    'FETP-3VCJ',
+    'FETP-7O3P',
+    'FETP-SVT1',
+    'FETP-1H38',
+    'FETP-AWYS',
+    'FETP-ISN8',
+    'FETP-0L1U',
+    'FETP-XYMR',
+    'FETP-14CZ',
+    'FETP-AN0I',
+    'FETP-P7S9',
+    'FETP-POPN',
+    'FETP-S72P',
+    'FETP-R8WC',
+    'FETP-36WM',
+    'FETP-MBNV',
+    'FETP-SKGQ',
+    'FETP-SMZS',
+    'FETP-5QZG',
+    'FETP-HTR7',
+    'FETP-S9XO',
+    'FETP-0WX7',
+    'FETP-TODV',
+    'FETP-END6',
+    'FETP-FL45',
+    'FETP-4KNV',
+    'FETP-XPV5',
+    'FETP-PLE1',
+    'FETP-A5NM',
+    'FETP-UOVP',
+    'FETP-P1YX',
+    'FETP-E7SF',
+    'FETP-FYML',
+    'FETP-Z2WP',
+    'FETP-COF9',
+    'FETP-GUND',
+    'FETP-HFQU',
+    'FETP-D8DS',
+    'FETP-WAUD',
+    'FETP-SU1C',
+    'FETP-LNZW',
+    'FETP-QBXC',
+    'FETP-XDRP',
+    'FETP-Y1O2',
+    'FETP-U9TW',
+    'FETP-IO9O',
+    'FETP-1D1V',
+    'FETP-I96L',
+    'FETP-S3CU',
+    'FETP-MSH3',
+    'FETP-JPT6',
+    'FETP-RW3S',
+    'FETP-KHGF',
+    'FETP-9SFK',
+    'FETP-5VLT',
+    'FETP-IHBP',
+    'FETP-78NT',
+    'FETP-2G3X',
+    'FETP-WZIN',
+    'FETP-LT0O',
+    'FETP-ZUIX',
+    'FETP-3J2M',
+    'FETP-PWEI',
+    'FETP-FTHC',
+    'FETP-YJDU',
+    'FETP-C04B',
+    'FETP-YLUK',
+    'FETP-2BY0',
+    'FETP-KJZP',
+    'FETP-L5ZX',
+    'FETP-832V',
+    'FETP-56E3',
+    'FETP-ZAEL',
+    'FETP-OOJO',
+    'FETP-TG6Z',
+    'FETP-ML91',
+    'FETP-B08Y',
+    'FETP-QLGW',
+    'FETP-P4N7',
+    'FETP-EJQ2',
+    'FETP-QPQN',
+    'FETP-JCOT',
+    'FETP-Y3TO',
+    'FETP-3F0Y',
+    'FETP-M5BJ',
   ];
-  for (const entityCode of entityCodes) {
+  for (let i = 0; i < originalEntityCodes.length; i++) {
+    await db.runSql(`
+    UPDATE entity 
+    SET code = '${newEntityCodes(i)}'
+    where code = '${originalEntityCodes(i)}';
+
+    UPDATE clinic 
+    SET code = '${newEntityCodes(i)}'
+    where code = '${originalEntityCodes(i)}';
```

Comment:
```suggestion
```
I think those are all individuals so we don't need to update clinic table

Reply:
Yeah you are right. Thanks Chris, I will delete these codes.